src/learn/ExperienceModule.tsx

```diff
@@ -490,7 +517,7 @@ class ExperienceModule extends Component<ExperienceModuleProps, ExperienceModule
               if (
                 reason !== "backdropClick" &&
                 reason !== "escapeKeyDown" &&
-                this.props.user.learningPurchase === false
+                this.props.user.learningPurchase === true
```

Comment:
Not sure if you maybe forgot to flip this back?

Reply:
yes I did, thought I had gotten them both, but I guess not

Reply:
done